### ISSUE TITLE
Doc Accessibility ACC-011: Insufficient visibility of the back to top link (moderate)

### ISSUE THREAD

**Issue by tfrancart** [sparna-git/shacl-play#329]

**Comment by tfrancart** [comment]
Reported on ReSpec at https://github.com/speced/respec/issues/5337
